src/main/java/org/zalando/nakadi/domain/SubscriptionBase.java

```diff
@@ -23,6 +24,7 @@
         CURSORS
     }
 
+    private String id;
```

Comment:
I don't exactly get it. Now both `Subscription` and `SubscriptionBase` have `id` field. What is the idea behind that?

Reply:
added in the commit